Header header

uint8 radar_num
uint8 num_objects
float64[] accel_x
float64[] vel_x
float64[] pos_x
float64[] pos_y
float64[] exist_prob
bool[] valid
